recipes/serpent/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.12" %}
+
+package:
+  name: serpent
```

Comment:
I'm not attached to either solution. So, if we are ok with it as is, let's leave it.